host-controller/src/main/java/org/jboss/as/host/controller/HostControllerEnvironment.java

```diff
@@ -307,7 +308,7 @@ public HostControllerEnvironment(Map<String, String> hostSystemProperties, boole
                 // Give up
                 qualifiedHostName = "unknown-host.unknown-domain";
             } else {
-                qualifiedHostName = qualifiedHostName.trim().toLowerCase();
+                qualifiedHostName = qualifiedHostName.trim().toLowerCase(Locale.ENGLISH);
```

Comment:
@soul2zimate this is user input so I think the system locale is appropriate here.

Reply:
Thank you for the review, I updated it to `Locale.getDefault()`